package.json

```diff
@@ -90,5 +96,19 @@
     "ts-patch": "^2.1.0",
     "typescript": "^4.9.3",
     "typescript-transform-paths": "^3.4.6"
+  },
+  "dependenciesMeta": {
+    "buildcheck@0.0.6": {
+      "unplugged": true
+    }
+  },
+  "resolutions": {
+    "@polkadot/api": "10.11.1",
+    "@polkadot/api-augment": "10.11.1",
+    "@polkadot/keyring": "12.6.1",
+    "@polkadot/util": "12.6.1",
+    "@polkadot/util-crypto": "12.6.1",
+    "@polkadot/wasm-crypto": "7.2.1",
+    "@polkadot/x-randomvalues": "12.6.1"
```

Comment:
what does this serve? As I understand what `resolutions` do, they are forcing specific versions of sub-dependencies, in the format of `<dependency>/<subdependency>: <subdependency_version>`, but here, there are only top-level dependencies listed

Reply:
When I've worked with `resolutions`, it's been enough to declare them at the top level of the workspace, unlike npm `overrides`. Declaring them once at the top level is meant to affect nested versions according to the [original RFC](https://github.com/yarnpkg/rfcs/blob/master/implemented/0000-selective-versions-resolutions.md#example).

The reason I had to do this is the auto incrementing you get when putting `^12.6.1` for polkadot versions causes the TypeScript types to go out of sync. `ApiPromise` and other things like `WeightV2` suddenly do not match. This is a constant source of bother with PJS which is why we always fix our versions. 

I'm currently upgrading our packages to `12.6.2`, which is why I haven't updated this PR for a bit. I should hopefully get the latest and greatest PJS into this PR next week.

Reply:
PJS versions in this PR will now be latest so I'll remove resolutions.

Reply:
Resolutions removed.